### ISSUE TITLE
Mountain River

### ISSUE THREAD

**Issue by cetooley** [Templarian/MaterialDesign#4469]
Hello!

We are using this in our project already but I figure it would be cool to include it in this icon set - there might be others out there who want a mountain-river for some reason.

I have:

- [x] Searched open issues to make sure there isn't a request for this icon.
- [x] Followed the [material guidelines](https://material.io/design/iconography/system-icons.html). (I hope!!)

## Preview
![image](https://user-images.githubusercontent.com/33706076/66229228-3890b000-e696-11e9-9c31-2bb91b29732a.png)

![image](https://user-images.githubusercontent.com/33706076/66229244-43e3db80-e696-11e9-91a1-b3e4a42ec74e.png)

## Zip Download
[mountain-river.zip](https://github.com/Templarian/MaterialDesign/files/3691717/mountain-river.zip)

**Comment by MrGrigri** [comment]
I'm afraid that it's lost at 1x

**Comment by JamesCoyle** [comment]
I don't like the perspective it uses but I'm not sure how you would show it without perspective.

**Comment by goyney** [comment]
What is the use case that the river is necessary for?

**Comment by cetooley** [comment]
> What is the use case that the river is necessary for?

In our case it is to denote a river source from the mountains - I am fine with this not being included, we use it in house but it's not a huge deal if it's not in the font set. We can use SVG as a font it just takes a bit of jiggering with css to match the other icons we use.

**Comment by goyney** [comment]
I think the problem here is the same we struggle with past requests for things such as pond, ocean, lake, etc. These things are super hard to represent and differentiate in the limited space we have while following Material guidelines. River is a little bit different than a pool-like body of water, but there are still issues with perspective and intent.

For all of those reasons, I'm going to recommend you use `water` or `waves` to denote a water source contextually, just as we have recommended for past requests. We've recently added some new `waves` icons to indicate water flow as well that might be useful. However, as this request stands, I don't feel that it is a good fit in the library at this time. Thank you for your suggestion.
